NickvisionTubeConverter.Shared/org.nickvision.tubeconverter.metainfo.xml.in

```diff
@@ -55,6 +55,7 @@
         <p>- Added the option to prevent suspend while using Parabolic</p>
         <p>- Improved bitrate used for audio-only downloads with the best quality</p>
         <p>- Parabolic will now prefer videos with the h264 codec when downloading in the mp4 format. If space is a concern, users are recommended to download in the webm format which uses the vp* codec</p>
+        <p>- If overwriting files is off and a download's file name exists, a numbered prefix will be attached to the end of the file name to prevent error</p>
```

Comment:
```suggestion
        <p>- If overwriting files is off and a download's file name exists, a numbered suffix will be attached to the end of the file name to prevent error</p>
```